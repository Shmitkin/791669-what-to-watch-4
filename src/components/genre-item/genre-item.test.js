import React from "react";
import renderer from "react-test-renderer";
import GenreItem from "./genre-item.jsx";

it(`Should GenreItem render correctly`, () => {
  const tree = renderer
    .create(
        <GenreItem
          genre = {`Strange Genre`}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
