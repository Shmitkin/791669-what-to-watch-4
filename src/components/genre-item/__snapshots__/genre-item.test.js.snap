// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should GenreItem render correctly 1`] = `
<li
  className="catalog__genres-item"
>
  <a
    className="catalog__genres-link"
    href="#"
  >
    Strange Genre
  </a>
</li>
`;
